flink-connectors/flink-connector-kinesis/pom.xml

```diff
@@ -204,6 +224,12 @@ under the License.
 									<include>com.amazonaws:*</include>
 									<include>com.google.protobuf:*</include>
 									<include>org.apache.httpcomponents:*</include>
+									<include>software.amazon.awssdk:*</include>
+									<include>software.amazon.eventstream:*</include>
+									<include>software.amazon.ion:*</include>
+									<include>org.reactivestreams:*</include>
+									<include>io.netty:*</include>
+									<include>com.typesafe.netty:*</include>
```

Comment:
I guess you'll have to update the `NOTICE` file of the Kinesis connector as well. (The FAQ section here should give you some pointers: https://cwiki.apache.org/confluence/display/FLINK/Licensing )

Reply:
Ok thanks I will take a look and update that

Reply:
@rmetzger I have updated the NOTICE file as requested. All of the new dependencies are Apache 2.0 except reactive streams is creative commons. Is that ok to bundle? I could not shade that one?

Reply:
Thanks a lot. We can include this dependency as well (see: https://www.apache.org/legal/resolved.html#handling-public-domain-licensed-works)